sylt-parser/src/parser.rs

```diff
@@ -275,6 +319,17 @@ impl<'a> Context<'a> {
         &self.peek().0
     }
 
+    fn tokens_forward<const N: usize>(&self) -> [Token; N] {
```

Comment:
This should be used when lookahead is needed. Sadly even if we don't use this it will still work when there aren't any comments.

That is, we want to do this: https://github.com/FredTheDino/sylt-lang/pull/344/commits/a4cb50e06fbae803dd282d783aa9ac65272524f2

Reply:
Maybe name it `tokens_lookahead` instead?

Reply:
Go for it - but do it in another PR

Reply:
I mean, it was introduced in this PR.

Reply:
Then go for it!